recipes/libsoup/libsoup.inc

```diff
@@ -0,0 +1,21 @@
+DESCRIPTION = "libsoup is an HTTP client/server library for GNOME."
+HOMEPAGE = "LGPLv2"
```

Comment:
```suggestion
LICENSE = "LGPL-2.0-only"
```
or is it LGPL-2.0-or-later ?

Reply:
Both libsoup and gssdp was LGPL-2.0-or-later